test/test_datasets.py

```diff
@@ -111,10 +111,10 @@ def test_imagefolder(self):
 
     def test_imagefolder_empty(self):
         with get_tmp_dir() as root:
-            with self.assertRaises(RuntimeError):
+            with self.assertRaises(FileNotFoundError):
```

Comment:
This BC breaking if someone relies on `DatasetFolder` or `ImageFolder` to raise a `RuntimeError` if no samples are found. Since the error type was never documented anywhere, I think this is fine.

Reply:
I'd say this is fine but might be worth checking if it breaks something in FBCODE.